src/main/js/ephox/imagetools/util/ImageResult.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * ImageResult.js
+ *
+ * Released under LGPL License.
+ * Copyright (c) 1999-2015 Ephox Corp. All rights reserved
+ *
+ * License: http://www.tinymce.com/license
+ * Contributing: http://www.tinymce.com/contributing
+ */
+
+define("ephox/imagetools/util/ImageResult", [], function() {
+
+    /**
+     * Creates data structure that will hold image data simultaneously as blob and as dataUri
+     *
+     * @method create
+     * @static
+     * @param {Blob} blob
+     * @param {String) dataUri
+     */
+    function create(blob, uri) {
+        return {
```

Comment:
In Textboxio, we tend to prefer an object of functions rather than just strings, so that if the developer uses the wrong property name, it throws an error rather than returns undefined (silently). We refer to it as a Struct. Talk to spocke about whether or not we want to do this here.